src/AuthorizedSocketConnection.js

```diff
@@ -46,7 +45,6 @@ export default class AuthorizedSocketConnection<TContext, TCredentials> {
   config: AuthorizedSocketOptions<TContext, TCredentials>;
```

Comment:
should this not extend `SocketConnection`?

Reply:
Yes, I was just sort of playing with it, not sure I wanna do that split tho. I'll clean up the files when I'm done